### DISCUSSION TITLE
[HSRC] USB3.0 expansion port

### DISCUSSION THREAD

**Discussion by okdhryk** [HumanSupportRobot/QandA#59] [Q&A]
The USB3.0 expansion port  is available in the HSRC, but are you using it?
There is no response when a device is connected to this port.

**Comment by hsr-admin** [comment]
Is the USB 3.0 expansion port you mentioned the port shown in the figure below?
If so, it is dedicated to TX2, so I would expect some kind of response from TX2.

<img width="408" alt="usb3 0port" src="https://github.com/HumanSupportRobot/QandA/assets/103644635/fc1b31f1-b87f-4a8f-9bd7-cc3b8f2867f7">

**Reply by okdhryk** [reply]
<img width="323" alt="スクリーンショット 2023-06-28 7 46 44" src="https://github.com/HumanSupportRobot/QandA/assets/498658/565cb28d-1be2-4928-a111-b552010fdc1f">

USB port attached to the back of the head.
Is this also for TX2?

**Reply by hsr-admin** [reply]
The usb3.0 expansion port on the head is on the CPU side, but when I checked it with my HSRC, it did not respond to anything.

**Reply by okdhryk** [reply]
This port has a 5 V power supply but no signal.
Two HSRCs (No.26 and No.28), both in the same condition.

**Reply by hsr-admin** [reply]
I installed and tried the standard ubuntu-ja-22.04-desktop-amd64.iso with the same results.

**Reply by hsr-admin** [reply]
Just to be sure, I checked the 23.04 release version for response to TX2.
UBS3.0 expansion port on head: no response
(UBS3.0 expansion port on access panel: responsive)

**Reply by hsr-admin** [reply]
We have confirmed the operating conditions of the USB3.0 port on the head and report the following.

- **USB memory stick compatible with USB3**
Some are recognized and some are not.

- **Mouse**
Not recognized

- **Real sense**
Recognized

So, depending on the device, there are some that can be used and some that cannot be used.
